components/typography/index.css

```diff
@@ -13,6 +13,267 @@ governing permissions and limitations under the License.
+@mixin typographyTShirtSizes .spectrum-BodyXXXL {}
+@mixin typographyTShirtSizes .spectrum-BodyXXL {}
+@mixin typographyTShirtSizes .spectrum-BodyXL {}
+@mixin typographyTShirtSizes .spectrum-BodyL {}
+@mixin typographyTShirtSizes .spectrum-BodyM {}
+@mixin typographyTShirtSizes .spectrum-BodyS {}
+@mixin typographyTShirtSizes .spectrum-BodyXS {}
+
+/* Detail / former subheadings t-shirt sizes */
+@mixin typographyTShirtSizes .spectrum-DetailXL {}
+@mixin typographyTShirtSizes .spectrum-DetailL {} 
+@mixin typographyTShirtSizes .spectrum-DetailM {} 
+@mixin typographyTShirtSizes .spectrum-DetailS {} 
+
+/* @TODO does not exists for strong or heavy for Details. Will be added in the future */
+/* 
+@mixin typographyTShirtSizes .spectrum-DetailXL--strong, detail-strong-XL { background-color: orange;}
+@mixin typographyTShirtSizes .spectrum-DetailL--strong, detail-strong-L {}
+@mixin typographyTShirtSizes .spectrum-DetailM--strong, detail-strong-M {}
+@mixin typographyTShirtSizes .spectrum-DetailS--heavy, detail-heavy-S {} */
+
+@mixin typographyTShirtSizes .spectrum-DetailXL--light, detail-light-XL {}
+@mixin typographyTShirtSizes .spectrum-DetailL--light, detail-light-L {}
+@mixin typographyTShirtSizes .spectrum-DetailM--light, detail-light-M {}
+@mixin typographyTShirtSizes .spectrum-DetailS--light, detail-light-S {}
+
+
+/* Code t-shirt sizes */
+@mixin typographyTShirtSizes .spectrum-CodeXL, code-XL, true {
+  font-family: var(--spectrum-code-xl-text-font-family);
+}
+@mixin typographyTShirtSizes .spectrum-CodeL, code-L, true {
+  font-family: var(--spectrum-code-l-text-font-family);
+}
+@mixin typographyTShirtSizes .spectrum-CodeM, code-M, true {
+  font-family: var(--spectrum-code-m-text-font-family);
+}
+@mixin typographyTShirtSizes .spectrum-CodeS, code-S, true {
+  font-family: var(--spectrum-code-s-text-font-family);
+}
+@mixin typographyTShirtSizes .spectrum-CodeXS, code-XS, true {
+  font-family: var(--spectrum-code-xs-text-font-family);
+}
+
+
+.spectrum-Typography {
+  @mixin typographyTShirtMargins .spectrum-HeadingXXXL {}
+  @mixin typographyTShirtMargins .spectrum-HeadingXXL {}
+  @mixin typographyTShirtMargins .spectrum-HeadingXL {}
+  @mixin typographyTShirtMargins .spectrum-HeadingL {}
+  @mixin typographyTShirtMargins .spectrum-HeadingM {}
+  @mixin typographyTShirtMargins .spectrum-HeadingS {}
+  @mixin typographyTShirtMargins .spectrum-HeadingXS {}
+  @mixin typographyTShirtMargins .spectrum-HeadingXXS {}
+
+  @mixin typographyTShirtMargins .spectrum-BodyXXXL {}
+  @mixin typographyTShirtMargins .spectrum-BodyXXL {}
+  @mixin typographyTShirtMargins .spectrum-BodyXL {}
+  @mixin typographyTShirtMargins .spectrum-BodyL {}
+  @mixin typographyTShirtMargins .spectrum-BodyM {}
+  @mixin typographyTShirtMargins .spectrum-BodyS {}
+  @mixin typographyTShirtMargins .spectrum-BodyXS {}
+}
+
+
+/* T-Shirt sizes for .spectrum-Article */
+.spectrum-Article {
+  font-family: var(--spectrum-body-xl-text-font-family);
```

Comment:
The font family here would need to have been `body-serif-xl-text-font-family` since the old "Article" paradigm = the new Serif option. Again, this wrapper class may not be appropriate, but all `serif` classnames should reference a font-family token from a `serif` DNA option